code/modules/mob/living/simple_animal/hostile/ooze.dm

```diff
@@ -41,6 +41,8 @@
 	ADD_TRAIT(src, TRAIT_VENTCRAWLER_ALWAYS, INNATE_TRAIT)
 	AddElement(/datum/element/content_barfer)
 
+	grant_multiple_actions(get_innate_actions())
```

Comment:
If the default behavior is "grants no actions", probably cleaner to just make the proc itself do it

IE: replace `grant_multiple_actions(get_innate_actions())` -> `initialize_actions()`, which subtypes can override

Reply:
ehhhh i built the proc to be nullsafe so i figure we should use the whole proc instead of confangling it further

Reply:
Goofy ahh, it's con-fanged anyways

Reply:
there's no winners. if it would make you happy i could store the subtype-specific actions in a list instead of using a getter proc but i like the getter proc